js/bootstrap-datepicker.js

```diff
@@ -1581,6 +1581,9 @@
 		});
 		delete options.inputs;
 
+		this.keepEmptyValues = options.keepEmptyValues || false;
```

Comment:
The default value should be adde to the defaults array, so we can keep all default values in one place